### ISSUE TITLE
Unknown format for Gemini: {"role":"user","parts" -- in promptfoo redteam eval

### ISSUE THREAD

**Issue by RubensZimbres** [promptfoo/promptfoo#1917]
**Describe the bug**
I successfully run promptfoo redteam init + promptfoo redteam generate for Google VertexAI Gemini pro. However when I run promptfoo eval, an unknown format for Gemini shows up.

Also, promptfoo gives errors when I add Indirect Prompt Injection and Prompt extraction. If I remove these from redteam init, it fails on the `eval`

**To Reproduce**
nvm use 20.10.0
npm install -g promptfoo
promptfoo redteam init (+Y generate)
promptfoo redteam eval

See error:

```
Unknown format for Gemini: {"role":"user","parts":{"text":"You are a travel agent specialized in budget trips to Europe\n\nUser q}
```
My redteam.yaml looks like this:

```
description: My first red team
prompts:
  - |+
    You are a travel agent specialized in budget trips to Europe.

    User query: {{prompt}}
```

My promptfooconfig.yaml looks like this:
```
description: "My first red team"

prompts:
  - "You are a travel agent specialized in budget trips to Europe.\n\nUser query: {{prompt}}\n\n"
  
targets:
  - vertex:gemini-pro
```

**Expected behavior**
I was expecting to run the redteam eval without issues

**Screenshots**
![Screenshot from 2024-10-17 10-20-51](https://github.com/user-attachments/assets/e308e440-9b29-4a7b-a913-a8eb49789b09)

**System information (please complete the following information):**

- Promptfoo version: 0.93.3 running on nvm 20.10.0 in Ubuntu 22.04, VSCode

**Comment by typpo** [comment]
Hey @RubensZimbres, this warning _should_ be safe to ignore.  It was triggering erroneously and is removed in bdf01b6499ec0738861ddf9bb3f970baaf1fb676

You can test a smaller number of inputs e.g. with `promptfoo redteam eval -n 2` (first 2) just to double check that everything is going through ok.

Let me know if you run into any issues with the final result.

**Comment by mldangelo** [comment]
Hi @RubensZimbres,

The warning for the Gemini format has been fixed in version [0.94.0](https://github.com/promptfoo/promptfoo/releases/tag/0.94.0). Let us know if you encounter any further issues with the final results!
